.github/workflows/on-push.yml

```diff
@@ -0,0 +1,79 @@
+name: Build master on push
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  BuildAndTest:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        configuration: ['Debug', 'Release']
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        include:
+          - os: windows-latest
+            libframework: net45
+          - os: ubuntu-latest
+            libframework: netstandard2
+          - os: macos-latest
+            libframework: netstandard2
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Setup .NET Core
+      if: matrix.libframework == 'netstandard2'
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 2.2.108
```

Comment:
Is 2.2.108 the appropriate version to use at this point? (I mean, I think it's one of the versions that's out of support, and I think the other PR for the nullability changes would require a v3 SDK for C#8 support)

Reply:
Yeah, I think using the latest stable would be more appropriate